src/tests/multihost/alltests/test_automount.py

```diff
@@ -510,3 +510,44 @@ def test_009_maps_after_coming_online(self, multihost, add_nisobject):
         cmd = 'dnf remove -y firewalld'
         multihost.client[0].run_command(cmd, raiseonerr=True)
         assert cmd2 == 0
+
+    @pytest.mark.parametrize('add_nisobject', ['/export'], indirect=True)
+    @pytest.mark.tier1
+    def test_010_delay_in_unknown_mnt_pt_lookup_error(self, multihost,
+                                                      add_nisobject):
+        """
+        :title: IDM-SSSD-TC: LDAP-Provider: Automount: autofs lookups for
+         unknown mounts are delayed for 50s
+        :id: 74cf1842-a8a0-4b6c-9670-82d73eb8ec6d
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2013218
+        :customerscenario: true
+        :steps:
+          1. Edit sssd.conf and specify autofs_provider = ldap and restart
```

Comment:
this part cant be seen in code

Reply:
Latest job is showing above part of docstring.